Tests/XcodeprojTests/GenerateXcodeprojTests.swift

```diff
@@ -43,8 +43,34 @@ class GenerateXcodeprojTests: XCTestCase {
         }
 #endif
     }
-    
+
+    func testXcconfigOverrideValidatesPath() {
+#if os(macOS)
+        mktmpdir { dstdir in
+            func dummy() throws -> [Module] {
+                return [try SwiftModule(name: "DummyModuleName", sources: Sources(paths: [], root: dstdir))]
+            }
+
+            let projectName = "DummyProjectName"
+            let dummyPackage = Package(manifest: Manifest(path: dstdir, url: dstdir.asString, package: PackageDescription.Package(name: "Foo"), products: [], version: nil), path: dstdir, modules: [], testModules: [], products: [])
+            let graph = PackageGraph(rootPackage: dummyPackage, modules: try dummy(), externalModules: [])
```

Comment:
After the inmemory change this can be simplified to : 
```swift
let fs = InMemoryFileSystem(emptyFiles:
    "/Bar/bar.swift",
)
let graph = try loadMockPackageGraph([
    "/Bar": Package(name: "Bar")
], root: "/Bar", in: fs)
```